README.md

```diff
@@ -71,83 +38,52 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
 
-## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
+## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
+You need to have a code editor pre-installed (preferably Vs Studio Code Editor). You may also need to install Git Bash terminal but if you're using Vs Code Studio you can use its terminal.
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
 
 ### Setup
 
 Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+1. Create a directory from which you will clone your repository.
+2. open the newly created directory and clone the repository using the git clone command
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
+1. install WebHint: npm install --save-dev hint@7.x
+2. install Stylelint: npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+3. install live Server from the extensions section on Vs Code Studio if using this code editor
 
 ### Usage
 
 To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
+open the cloned repository in your code editor
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+To run tests and check for errors:
+-After installing the required linter, npx stylelint "**/*.{css,scss}"
+-To fix CSS or SCSS linters error: npx stylelint "**/*.{css,scss}" --fix
+-Run your code on web browser
 
 ### Deployment
 
 You can deploy this project using: github pages
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile: